homeassistant/components/home_connect/const.py

```diff
@@ -9,6 +10,24 @@
 DOMAIN = "home_connect"
 
 
+class ApplianceType(StrEnum):
+    """Appliance type constants."""
+
+    CLEANING_ROBOT = "CleaningRobot"
+    COFFEE_MAKER = "CoffeeMaker"
+    COOKING = "Cooking"
+    DISHWASHER = "Dishwasher"
+    DRYER = "Dryer"
+    FRIDGE_FREEZER = "FridgeFreezer"
+    FREEZER = "Freezer"
+    HOOD = "Hood"
+    OVEN = "Oven"
+    REFRIGERATOR = "Refrigerator"
+    WARMMING_DRAWER = "WarmingDrawer"
+    WASHER = "Washer"
+    WASHER_DRYER = "WasherDryer"
```

Comment:
Maybe this could be added to aiohomeconnect models

Reply:
Yes